### ISSUE TITLE
MP? for "lung ciliated cell differentiation"

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#12378]
Hi Paola

I was surprised to see that "lung ciliated cell differentiation" (GO:0061141) does not have a relationship to "multi-ciliated epithelial cell differentiation" (GO:1903251).

I see in the definition of "lung ciliated cell differentiation" that it says "a motile cilium". However, I am under the impression that lung epithelial cells, as with other types of epithelial cells that are involved in moving fluids, are multiciliated cells. 

term: "lung ciliated cell differentiation"
Def: The process in which a relatively unspecialized cell acquires specialized features of a lung ciliated cell. A lung ciliated cell is a specialized lung epithelial cell that contains a motile cilium for moving substances released from lung secretory cells.

Not urgent for me, but probably worth checking on.

thanks,

-Karen

@paolaroncaglia

**Comment by cmungall** [comment]
[CL:1000271](http://purl.obolibrary.org/obo/CL_1000271) ! lung ciliated cell 

```
  is_a [CL:0000000](http://purl.obolibrary.org/obo/CL_0000000) ! cell
   is_a [CL:0000003](http://purl.obolibrary.org/obo/CL_0000003) ! native cell
    is_a [CL:0000064](http://purl.obolibrary.org/obo/CL_0000064) ! ciliated cell
     is_a [CL:1000271](http://purl.obolibrary.org/obo/CL_1000271) ! lung ciliated cell  
```

looks like the incompleteness is on the side of CL. Can you file here: https://github.com/obophenotype/cell-ontology/issues

thanks

**Comment by krchristie** [comment]
CL issue is here: https://github.com/obophenotype/cell-ontology/issues/432 

GO definition may still need revision.

-Karen

**Comment by paolaroncaglia** [comment]
Hi @krchristie ,

I commented on the CL ticket to see if any of the editors there can take a look.
In the meantime, I modified the def of "lung ciliated cell differentiation" (GO:0061141) to reflect the multi-ciliated nature:
"...A lung ciliated cell is a specialized lung epithelial cell that contains cilia for moving substances..."
When the CL issue is dealt with, I will fix the parentage of "lung ciliated cell differentiation" (GO:0061141). In the meantime, I'll set to pending.
